Source/Data/BPMFMappings.txt

```diff
@@ -139253,3 +139253,4 @@
 龜鑑 ㄍㄨㄟ ㄐㄧㄢˋ
 龜頭 ㄍㄨㄟ ㄊㄡˊ
 龜鱉目 ㄍㄨㄟ ㄅㄧㄝ ㄇㄨˋ
+烤地瓜 ㄎㄠˇ ㄉㄧˋ ㄍㄨㄚ
```

Comment:
issue: please ensure the mapping file is sorted.

`LC_ALL=c sort -o BPMFMappings.txt BPMFMappings.txt`

Ref: https://github.com/openvanilla/McBopomofo/wiki/%E8%A9%9E%E5%BA%AB%E9%96%8B%E7%99%BC%E8%AA%AA%E6%98%8E